platform/darwin/src/MGLAnnotation.h

```diff
@@ -36,6 +41,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, copy, nullable) NSString *subtitle;
 
+#if TARGET_OS_IPHONE
+
+@property (nonatomic) BOOL canShowCallout;
```

Comment:
Come to think of it, these view-related properties don’t belong in a model protocol if we’re going to respect a conventional MVC pattern. MapKit places this and other callout-related properties on MKAnnotationView, which prevents the developer from customizing the callout view for custom model objects that are used as point annotations. Once #4801 lands, it may be worth moving these properties over to MGLAnnotationView.